canonical-web-and-design/maas-ui
# feat(ui): build Other images section on images page

## Done

- Built "Other images" section on images page
- Moved some files around to be more clear what components are reusuable (`ImagesTable`) and what are specific for a section (`UbuntuImageSelect`)

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to /MAAS/r/images and check there is an "Other images" section
- Check that you can add and remove "other images", noting that you will need to refresh the page to get updates on any changes

## Fixes

Fixes canonical-web-and-design/app-squad#82

## Screenshot

![2021-06-17_16-05](https://user-images.githubusercontent.com/25733845/122340649-e08eff80-cf85-11eb-9da0-620054c9b8ae.png)
